client/src/components/contact-details/index.vue

```diff
@@ -60,6 +60,18 @@
 
             </md-list-item>
 
+            <md-list-item>
+
+              <div>
+                <md-chips v-model="labels_names" md-static></md-chips>
+                <div class="md-list-item-text">
+                  <span></span>
+                  <span>Labels</span>
+                </div>
+              </div>
+
+            </md-list-item>
+
             <md-divider></md-divider>
```

Comment:
Try to remove this <divider> as I looks bad 
<img width="1178" alt="screen shot 2018-08-15 at 11 15 27 am" src="https://user-images.githubusercontent.com/861869/44141606-dd725d04-a07d-11e8-978c-7a5a6360eaaa.png">